pm_icecon/bt/params/a2l1c.py

```diff
@@ -36,8 +32,8 @@
             weather_filter_params=WeatherFilterParams(
                 wintrc=82.71,
                 wslope=0.5352,
-                # wxlimt=23.34,  # Original value
-                wxlimt=21.7,  # Median of calc'd wx coefs
+                # Original value given in `ret_parameters_amsru2.f`
```

Comment:
Yes!  We want to document where this came from.

I wonder if that ret_parameters_amsru2.f is publicly available?

Reply:
Not that I'm aware of. It might be good to have some documentation noting where references like this originally came from. 

Ideally Goddard would publish their code so that we could point to it as a reference.